test/UI.Test/IdentityPageModelConventionTest.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Microsoft.AspNetCore.Mvc.ApplicationModels;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Identity.UI.Test
+{
+    public class IdentityPageModelConventionTest
```

Comment:
I would like to also add some functional tests with the custom identity user model but the currently test website that we have has a razor page which seems is for testing the overriding scenario (?) and uses the model as `IdentityUser` which doesn't work for my scenario (even if i have a difference Startup file having a custom identityuse). Any suggestions? 
Is it fine to create a separate website for this purpose? It feels like a overkill but we ideally want an end-to-end test.

Reply:
There is a separate website in Testing/WebSites